src/index.html

```diff
@@ -8,24 +8,270 @@
+            <!-- Card Body -->
+            <div>
+              <!-- first line -->
+              <div class="card_status">
+                <p>Low</p>
+                <p>On track</p>
+              </div>
+              <!-- second line -->
+              <div class="card_profile">
+                <p class="card_profile_initials">AA</p>
+                <p class="card_profile_date">27 Feb - 1 Mar</p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="cards">
+          <div class="card">
+            <!-- Card Header -->
+            <div class="card_header">
+              <h1>In Progress</h1>
+              <p><span>...</span></p>
+            </div>
+            <!-- Card Body -->
+            <div>
+              <!-- first line -->
+              <div class="card_status">
+                <p>Low</p>
+                <p>On track</p>
+              </div>
+              <!-- second line -->
+              <div class="card_profile">
+                <p class="card_profile_initials">AA</p>
+                <p class="card_profile_date">27 Feb - 1 Mar</p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="cards">
+          <div class="card">
+            <!-- Card Header -->
+            <div class="card_header">
+              <h1>In Progress</h1>
+              <p><span>...</span></p>
+            </div>
+            <!-- Card Body -->
+            <div>
+              <!-- first line -->
+              <div class="card_status">
+                <p>Low</p>
+                <p>On track</p>
+              </div>
+              <!-- second line -->
+              <div class="card_profile">
+                <p class="card_profile_initials">AA</p>
+                <p class="card_profile_date">27 Feb - 1 Mar</p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div></div>
       </div>
-      <div class="frame" id="inProgress">
-        <h1>In progress</h1>
+      <!-- In progress -->
+      <div class="frame" id="In_Progress">
+        <!-- Frame header -->
+        <div class="frame_header">
+          <h1>In Progress</h1>
+          <p><span>+</span><span>...</span></p>
+        </div>
+        <!-- Cards -->
+        <div class="cards">
+          <div class="card">
+            <!-- Card Header -->
+            <div class="card_header">
+              <h1>Learn TypeScript</h1>
+              <p><span>...</span></p>
+            </div>
+            <!-- Card Body -->
+            <div>
+              <!-- first line -->
+              <div class="card_status">
+                <p>Low</p>
+                <p>On track</p>
+              </div>
+              <!-- second line -->
+              <div class="card_profile">
+                <p class="card_profile_initials">AA</p>
+                <p class="card_profile_date">27 Feb - 1 Mar</p>
+              </div>
+            </div>
+          </div>
+        </div>
```

Comment:
you are misusing divs, take a look at tags like sections;
[https://dev.to/kenbellows/stop-using-so-many-divs-an-intro-to-semantic-html-3i9i](url)